packages/seacas/libraries/exodus/test/testwt-results.c

```diff
@@ -93,8 +93,8 @@ int main(int argc, char **argv)
   num_node_sets = 0;
   num_side_sets = 0;
 
-  error = ex_put_init(exoid, title, num_dim, num_nodes, num_elem, num_elem_blk, num_node_sets,
```

Comment:
No longer checking error code?  Never checked the error code?

Reply:
It was assigning, but never checking the error code before. Removed the assignment to avoid "unused variable warning"

The check in this case is later when it verifies that correct values exist in the file...